src/components/Header/Header.jsx

```diff
@@ -42,8 +80,12 @@ function Header() {
                         <input 
                             type="text"
                             placeholder="Search a Products"
+                            value={inputSearch}
+                            onChange={handleSearchFilterChange}
                         />
-                        <CloseLineIcon className={styles["close-icon"]} />
+                        {
+                            inputSearch !== "" && <CloseLineIcon className={styles["close-icon"]} onClick={clearInputSearch} />
```

Comment:
```suggestion
                            inputSearch && <CloseLineIcon className={styles["close-icon"]} onClick={clearInputSearch} />
```
Như này là được rồi nhé.
```
"" === false
"123" === true
```